roles/mons/tasks/main.yml

```diff
@@ -79,7 +79,7 @@
   command: ceph-authtool -n client.radosgw.gateway --cap osd 'allow rwx' --cap mon 'allow rwx' /etc/ceph/ceph.client.radosgw.keyring
 
 - name: Add RGW key to cluster
-  command: ceph -k /etc/ceph/ceph.client.admin.keyring auth add client.radosgw.gateway -i /etc/ceph/ceph.client.radosgw.keyring
```

Comment:
Not necessary for the first run, but if you rerun the provision step, this makes it not barf about keys not matching. Importing on each run.